src/contributing/prs.md

```diff
@@ -165,8 +165,12 @@ updating.
 
 The process of squashing is synonymous with `git rebase`, so when you are asked
 to squash it is also acceptable to: rebase the commit against the `master`
-branch at the same time; and force push the resulting rebased branch. *This is
```

Comment:
There is good terminology here I fear. All our repos are `master` (except those they've forked), so that remains the most widely understood choice IMHo.